manifests/istio-control/istio-discovery/templates/telemetryv2_1.5.yaml

```diff
@@ -31,10 +31,17 @@ spec:
               config:
                 configuration: envoy.wasm.metadata_exchange
                 vm_config:
+                  {{- if .Values.telemetry.v2.metadataExchange.enabledV8 }}
+                  runtime: envoy.wasm.runtime.v8
+                  code:
+                    local:
+                      filename: /etc/istio/extensions/metadata-exchange-filter.wasm
```

Comment:
where is this file extracted from?

Reply:
It's inside the docker image.

Reply:
Proxy pushes those wasm moduels in postsubmit and Istio pulls them when build proxy docker image.